README.md

```diff
@@ -1,51 +1,71 @@
+
+**اگر دوست دارین از این پروژه(نویسندگان اصلی آن ونه مترجمان) حمایت مالی کنید میتوانید با استفاده از [این لینک](donations.html) جایی که در آن میتوانید همه اهداف پروژه را نیز پیدا کنید ، اقدام کنید.
+همه حمایت های مالی به صورت کامل در چهارچوب توسعه و تکمیل این پروژه استفاده میشود. همه حمایت های مالی و نحوه خرج شدن آن ها در صفحه دونیشن ها ثبت و به عموم نمایش داده میشوند.** 
+
+**جایگزین های دانلود مستقیم:**
+
+**بزودی**
+
+**مشاهده راهنما در:**
+ [**(کلیک کنید)مرورگر شما**](guide.html)
+- <del>[PDF](export/guide.pdf)</del> (به صورت موقت در دسترس نیست)
+- <del>[OpenDocument (ODT)](export/guide.odt)</del> (به صورت موقت در دسترس نیست)
+- Raw <del>[Markdown](https://github.com/Anon-Planet/thgtoa/raw/master/guide.md)</del> (منابع در این حالت کار نمیکنند).
 
 Mirrors:
 - Tor Onion Mirror: <http://thgtoallkcxrdv37u6knsc3pumk6cq6lqmcqlw3j5vkmyahkxive4jyd.onion>
 
-The guide and all the files are also readily available on Archive.org and Archive.today:
+راهنما و تمام فایل ها به صورت آماده در Archive.org و Archive.today موجود میباشند:
 
 - Archive.org: <https://web.archive.org/web/https://anonymousplanet.org/>
 - Archive.today: <https://archive.ph/anonymousplanet.org/>
 - Archive.today over Tor: <http://archiveiya74codqgiixo33q62qlrqtkgmcitqx5u2oeqnmn5bpcbiyd.onion/anonymousplanet.org/>
 
 If you want to access/see the original/legacy project, please see the [legacy resources](legacy.html) page.
 
-If you want to see the changes between your PDF and the latest PDF, you could use one of these tools (we do not endorse those):
+اگر میخواهید تغییرات  بین اخرین pdf و pdf خودتان ببینید، میتوانید از یکی از این ابزار ها استفاده کنید (ما این ابزار ها را تایید نمیکنیم):
+
 
 - <https://tools.pdf24.org/en/compare-pdf>
 - <https://products.aspose.app/pdf/comparison>
 - <https://draftable.com/compare>
 
-If you want to compare an older ODT file with a newer one, use the LibreWriter compare features as explained here: <https://help.libreoffice.org/7.1/en-US/text/shared/guide/redlining_doccompare.html> <sup>[[Archive.org]](https://web.archive.org/wen/https://help.libreoffice.org/7.1/en-US/text/shared/guide/redlining_doccompare.html)</sup>)
+اگر میخواهید یک فایل ODT قدیمی تر را با جدیدتر مقایسه کنید، میتوانید از قابلیت Librewriter به شکلی که در لینک زیر توضیح داده شده است استفاده کنید:
+
+ <https://help.libreoffice.org/7.1/en-US/text/shared/guide/redlining_doccompare.html> <sup>[[Archive.org]](https://web.archive.org/wen/https://help.libreoffice.org/7.1/en-US/text/shared/guide/redlining_doccompare.html)</sup>)
+
+**اگر میخواهید یکپارچگی، امنیت و صحت فایل ها را بررسی کنید میتوانید به [چگونه](verify.html) مراجعه کنید.**
 
-**If you want to check the files for integrity, safety, authenticity, please refer to this ["How To"](verify.html).**
+اگر میخواهید، میتوانید مسائل مربوط به متن را در  بخش Github Issues این [این ریپازیتوری](https://github.com/Hamid-M-Mostafa/thgtoa) به زبان فارسی مطرح کنید.<br> از نقد، نظر و ایده های شما استقبال میشود! 
```

Comment:
This would need to be changed to the official GH repo, but other than that, I think this is good.